<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Importexcel</class>
 <widget class="QWidget" name="Importexcel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>考生导入</string>
  </property>
  <widget class="QLabel" name="label_File">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择需要导入的Excel：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_File">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_File">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_View">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>150</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>预览导入（只预览前10行）：</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectColumns</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Ok">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Top">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>首行为标题</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Count">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>200</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Note">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>预览表中各数据列名顺序为:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>考号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>姓名</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>班级</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="toolButton_Add">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>37</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>=&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Col1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>第一列为:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Col2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>第二列为:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Col3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>第三列为:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Col1_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Col3_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Col2_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_Clear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>37</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>重排</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Process">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
